website/docs/cloud-docs/api-docs/project-team-access.mdx

```diff
@@ -185,14 +224,24 @@ This POST endpoint requires a JSON object with the following properties as a req
-| `data.relationships.team.data.type`    | string |         | Must be `teams`.                                                 |
-| `data.relationships.team.data.id`      | string |         | The ID of the team to add to the project.                        |
+| Key path                                                  | Type    | Default | Description                                                                                        |
+|-----------------------------------------------------------|-------- |---------|----------------------------------------------------------------------------------------------------|
+| `data.type`                                               | string  |         | Must be `"team-projects"`.                                                                         |
+| `data.attributes.access`                                  | string  |         | The type of access to grant. Valid values are `read`, `write`, `maintain`, `admin`, or `custom`.     |
+| `data.relationships.project.data.type`                    | string  |         | Must be `projects`.                                                                                |
+| `data.relationships.project.data.id`                      | string  |         | The project ID to which the team is to be added.                                                   |
+| `data.relationships.team.data.type`                       | string  |         | Must be `teams`.                                                                                   |
+| `data.relationships.team.data.id`                         | string  |         | The ID of the team to add to the project.                                                          |
+| `data.attributes.project-access.settings`                 | string  | "read"  | If `access` is `custom`, the permission to grant for the project's settings. Can only be used when `access` is `custom`. Valid values include `read`, `update`, or `delete`.                                                   |
+| `data.attributes.project-access.teams`                    | string  | "none"  | If `access` is `custom`, the permission to grant for the project's teams. Can only be used when `access` is `custom`. Valid values include `none`, `read`, or `manage`.                                                   |
+| `data.attributes.workspace-access.runs`                   | string  | "none"  | If `access` is `custom`, the permission to grant for the project's workspaces' runs. Can only be used when `access` is `custom`. Valid values include `none`, `read`, `plan`, or `apply`.                                                    |
+| `data.attributes.workspace-access.sentinel-mocks`         | string  | "none"  | If `access` is `custom`, the permission to grant for the workspace's Sentinel mocks. Can only be used when `access` is `custom`. Valid values include `none`, or `read`.                                                     |
```

Comment:
The S in Sentinel is capitalized -- I double checked on the team access workspace docs.